src/System.Reflection.Metadata/ref/System.Reflection.Metadata.cs

```diff
@@ -1608,6 +1610,7 @@ public sealed partial class MetadataReader
         public System.Reflection.Metadata.AssemblyDefinition GetAssemblyDefinition() { throw null; }
         public System.Reflection.Metadata.AssemblyFile GetAssemblyFile(System.Reflection.Metadata.AssemblyFileHandle handle) { throw null; }
         public System.Reflection.Metadata.AssemblyReference GetAssemblyReference(System.Reflection.Metadata.AssemblyReferenceHandle handle) { throw null; }
+        public System.Reflection.AssemblyName GetAssemblyName() { throw null; }
```

Comment:
We should not expose it here too. I think it only clutters the API. `GetAssemblyDefinition().GetAssemblyName() ` is fine.